packages/date/src/index.js

```diff
@@ -395,7 +395,7 @@ export function dateI18n( dateFormat, dateValue = new Date(), gmt = false ) {
 /**
  * Check whether a date is considered in the future according to the WordPress settings.
  *
- * @param {(Date|string)} dateValue  Date object or string.
+ * @param {string} dateValue Date String or Date object in the Defined WP Timezone.
```

Comment:
So is it a string or an object? The current documentation here sends mixed messaging.